src/js/common/schemaform/save-in-progress/actions.js

```diff
@@ -200,6 +200,7 @@ export function fetchInProgressForm(formId, migrations, prefill = false, prefill
   //  redux store, but form.migrations doesn’t exist (nor should it, really)
   return (dispatch, getState) => {
     const trackingPrefix = getState().form.trackingPrefix;
+    // const imageEndpoint = getState().form.imageEndpoint;
```

Comment:
The plan here is to create `fetchFormApi` in `./api` and use imageEndpoint to set up something like:
```
    const formUrl = `${environment.API_URL}/v0/in_progress_forms/${formId}`;
    if (imageEndpoint) {
      const imageUrl = `${environment.API_URL}${imageEndpoint}`;

      Promise.all([formUrl, imageUrl].map(
        url => fetchFormApi(url, responseHandler, errorHandler)
      ))
        .then(processResponseAsUsual)
        .catch(errorHandler);
    } else {
      fetchFormApi(formUrl, responseHandler, errorHandler)
        .then(processResponseAsUsual)
        .catch(errorHandler);
    }

```

Reply:
I don't think I understand the purpose of that code.

Reply:
The purpose here is to retrieve the saved image file, as it is not being stored with the form data, correct? Or will we be able to use the same endpoint to retrieve this?